cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package controller
 
 import (
+	"encoding/json"
+	"fmt"
+	pb "github.com/kubeedge/kubeedge/pkg/apis/dmi/v1beta1"
```

Comment:
The import needs to be adjusted based on the sequence of the 'standard library, third-party library, and current project library'. Other places also need to be adjusted.